pegasus/forms/hoc_signup_2022.rb

```diff
@@ -0,0 +1,17 @@
+require pegasus_dir 'forms/hoc_signup_2020'
+
+# HOC Sign up form for 2022.
+# Most of the logic hasn't changed since the 2017 form, but the 2020 form added
+# support for "at_home" as an event type.
+class HocSignup2022 < HocSignup2020
+  def self.receipt(data)
+    country = data["hoc_event_country_s"].downcase unless data["hoc_event_country_s"].nil_or_empty?
+    if %w(ar bo cl co cr cu do ec gq gt hn mx ni pa pe pr py sv uy ve).include? country
+      'hoc_signup_2022_receipt_es'
+    elsif country == "br"
+      'hoc_signup_2022_receipt_pt'
+    else
+      'hoc_signup_2022_receipt_en'
+    end
```

Comment:
Nit:


I would try less hardcoded values
```
ES_COUNTRIES_RECEIPT = 'hoc_signup_2022_receipt_es'.freeze
PT_COUNTRIES_RECEIPT = 'hoc_signup_2022_receipt_pt'.freeze
EN_COUNTRIES_RECEIPT = 'hoc_signup_2022_receipt_en'.freeze

class HocSignup2022 < HocSignup2020
  def self.receipt(data)
    country = data["hoc_event_country_s"].downcase unless data["hoc_event_country_s"].nil_or_empty?

  es_countries = %w(ar bo cl co cr cu do ec gq gt hn mx ni pa pe pr py sv uy ve).split(' ')
  pt_countries = %w(br).split(' ')
  if es_country.includes? country
    ES_COUNTRIES_RECEIPT
  elsif pt_countrye.incluydes? country
    PT_COUNTRIES_RECEIPT
  elsif
  EN_COUNTRIES_RECEIPT
  end
```

This code wont run, adding it as reference :)

Reply:
Nice. I will leave as-is to be consistent with previous versions for now, but definitely agree it could be more readable and might revisit in any follow-up that we do.